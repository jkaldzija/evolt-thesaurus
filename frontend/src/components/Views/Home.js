import React from 'react';
import {Col, Row} from "reactstrap";
import SearchBox from "../SearchBox/SearchBox";
import AddWordView from "../AddWord/AddWordView";


// Set prop types
type HomeProps = {};

class Home extends React.Component<HomeProps> {
    componentDidMount() {
    }

    componentDidUpdate() {
    }

    render() {

        return (
            <Row>
                <Col sm={12} md={8} className={"mb-4"}>
                    <SearchBox/>
                </Col>
                <Col>
                    <AddWordView/>
                </Col>
            </Row>
        );
    };

}

// Set default props
Home.defaultProps = {};

export default Home;
