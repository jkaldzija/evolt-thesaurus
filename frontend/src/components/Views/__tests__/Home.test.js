import React from 'react';
import {shallow} from "enzyme";
import Home from "../Home";

describe('Search Box', function () {
    it('should render correctly', function () {
        const component = shallow(
            <Home onSubmit={() => {}}/>
        );
        expect(component).toMatchSnapshot();
    });

});
