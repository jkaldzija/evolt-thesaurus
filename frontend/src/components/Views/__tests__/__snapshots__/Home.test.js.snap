// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Box should render correctly 1`] = `
<Row
  tag="div"
>
  <Col
    className="mb-4"
    md={8}
    sm={12}
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <withRouter(SearchBox) />
  </Col>
  <Col
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <AddWordView />
  </Col>
</Row>
`;
